compiler-rt/lib/scudo/standalone/tests/combined_test.cpp

```diff
@@ -345,7 +345,7 @@ SCUDO_TYPED_TEST(ScudoCombinedTest, ReallocateLargeIncreasing) {
   // we preserve the data in the process.
   scudo::uptr Size = 16;
   void *P = Allocator->allocate(Size, Origin);
-  const char Marker = 0xab;
+  const char Marker = '\xab';
```

Comment:
Maybe just use 'A' here as well, rather than the hex escape prefix?

Reply:
Done.